utils/debug/registerCache.go

```diff
@@ -97,10 +97,10 @@ func (f *fileRegisterCache) Get(owner, key string) ([]byte, bool) {
 func (f *fileRegisterCache) Set(owner, key string, value []byte) {
 	valueCopy := make([]byte, len(value))
 	copy(valueCopy, value)
-	fmt.Println(hex.EncodeToString([]byte(owner)), hex.EncodeToString([]byte(key)), len(value))
+	ownerAddr := flow.BytesToAddress([]byte(owner))
+	fmt.Println(ownerAddr.Hex(), hex.EncodeToString([]byte(key)), len(value))
```

Comment:
is the println needed?

Reply:
I believe so. My understanding is that this code is used for the client side of the remote debugger. it could probably be refactored to use a proper logging lib, but I'll save that for another day